### DISCUSSION TITLE
Issue with Gateways connection

### DISCUSSION THREAD

**Discussion by Krishna1805** [migueesc123/PowerBIRESTAPI#176] [Announcements]
Gateways connection is not working and getting the error "Expression.Error: The column 'id' of the table wasn't found"

![image](https://user-images.githubusercontent.com/38653619/236770295-00f304b2-2a47-4332-b2e5-d00e6ca032f4.png)

**Comment by ImkeF** [comment]
Hi @Krishna1805 ,
this error message will appear when there are no gateways you have access to.
(Make sure you are connected to the right tenant...)
